api/src/main/java/org/eclipse/microprofile/rest/client/ext/ResponseExceptionMapper.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.microprofile.rest.client.ext;
+
+import javax.annotation.Priority;
+import javax.ws.rs.core.Response;
+import java.util.Optional;
+
+/**
+ * Converts an JAX-RS Response object into an Exception.
+ *
+ */
+public interface ResponseExceptionMapper {
+    int DEFAULT_PRIORITY = 100;
```

Comment:
Instead of hardcoding to 100, can we set it so the the default priority is `Priorities.USER` (which is 5000) or `Priorities.USER + 1` (or something higher than USER) for container-provided ResponseExceptionMappers.